tests/test_cli.py

```diff
@@ -68,14 +77,23 @@ def test_list_tool(self) -> None:
         UtilFunctions.run_list_tool(print_all=True)
         UtilFunctions.run_list_tool(ecu_services=["session_start"])
 
-    def test_find_tool(self) -> None:
+    def test_decode_tool(self) -> None:
+
+        UtilFunctions.run_decode_tool(data="3E00")
+        UtilFunctions.run_decode_tool(data="3e00")
+        UtilFunctions.run_decode_tool(data="3E 00", decode_data=True)
+        UtilFunctions.run_decode_tool(data="3E 00", ecu_variants=["somersault_lazy"])
+        UtilFunctions.run_decode_tool(data="3E 00")
 
-        UtilFunctions.run_find_tool(data="3E00", ecu_services=["session_start"])
-        UtilFunctions.run_find_tool(data="3e00", ecu_services=["session_start"], no_details=False)
-        UtilFunctions.run_find_tool(data="3E 00")
-        UtilFunctions.run_find_tool(data="3E 00", ecu_variants=["somersault_lazy"])
-        UtilFunctions.run_find_tool(data="3E 00", decode=True)
-        UtilFunctions.run_find_tool(data="3E 00", decode=True, relaxed_output=True)
+    def test_find_tool(self) -> None:
+        UtilFunctions.run_find_tool(service_names=["headstand"])
+        UtilFunctions.run_find_tool(service_names=["headstand"], allow_unknown_bit_lengths=True)
+        UtilFunctions.run_find_tool(
+            service_names=["headstand"], allow_unknown_bit_lengths=True, no_details=True)
+        # UtilFunctions.run_find_tool(service_names="3e00")
```

Comment:
avoid commented-out code if there's not a very good reason for it...

Reply:
true, I removed it